site/en/getstarted/standalone/install_standalone-docker.md

```diff
@@ -13,35 +13,42 @@ summary: Learn how to install Milvus stanalone with Docker Compose.
 
 {{tab}}
 
-## Download an installation file
+## Prerequisites
 
-[Download](https://github.com/milvus-io/milvus/releases/download/v{{var.milvus_release_tag}}/milvus-standalone-docker-compose.yml) `milvus-standalone-docker-compose.yml` directly or with the following command, and save it as `docker-compose.yml`.
+Check [the requirements](prerequisite-docker.md)  for hardware and software prior to your installation.
+
+## Download the Compose file
+
+[Download](https://github.com/milvus-io/milvus/releases/download/v{{var.milvus_release_tag}}/milvus-standalone-docker-compose.yml) `milvus-standalone-docker-compose.yml` and save it as `docker-compose.yml` manually, or with the following command.
 
 ```
 $ wget https://github.com/milvus-io/milvus/releases/download/v{{var.milvus_release_tag}}/milvus-standalone-docker-compose.yml -O docker-compose.yml
 ```
 
-
 ## Start Milvus
 
+In the same directory as the `docker-compose.yml` file, start up Milvus by running:
+
 ```shell
 $ sudo docker-compose up -d
 ```
 
+> If your system has Docker Compose V2 installed instead of V1, use `docker compose` instead of `docker-compose`. Check if this is the case with `$ docker compose version`. Read [here](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) for more information.
```

Comment:
Please use the <div class="alert note'> format. I am not sure if the Milvus doc website supports the "<" layout.